package.json

```diff
@@ -584,6 +584,18 @@
             {
                 "language": "brightscript",
                 "path": "./snippets.json"
+            },
```

Comment:
Note:
I wanted to have different snippets for brightscript and brighterscript, but I think because they are actually defined to use the same grammar, the snippets show up in both places. To be more clear, I just set them all to brightscript, although  kept the brighterscript specific ones in their own file

Reply:
That's fine for now. We can always split them out later. Thanks for keeping them in separate files.